addons/binding/org.openhab.binding.rotelra1x/src/main/java/org/openhab/binding/rotelra1x/internal/ConfigurationError.java

```diff
@@ -0,0 +1,14 @@
+package org.openhab.binding.rotelra1x.internal;
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.